test/new-e2e/examples/agentenv_npm_test.go

```diff
@@ -0,0 +1,40 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
Just realised, can you move this to `test/new-e2e/tests/npm` package and add npm team as owners ? Examples are meant for non team specific cases, they are not executed on the CI (yet). They are related to the workshop

Reply:
this is just the exercise
The actual tests will be in `test/new-e2e/npm` ndm seems doing the same

On the next [PR](https://github.com/DataDog/datadog-agent/pull/19449/files#diff-aa95d108dd22d77ac56d2b566ef3ff4185afc1c6f5102dfede87178393240696)

Reply:
It still makes sense to move it under `npm`, you can skip it with a flag so that you don't run it when targeting `npm` folder.

Reply:
This would also fix your current linter issue, if you move the example you can import the embed file within your file and skip exposing it publicly https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/329121543#L239

Reply:
I updated the comments
I think it make sense to export the npm.SystemprobeConfig as it could be used by others easily

Reply:
all moved to `.../tests/npm`